ldk/go/examples/filesystem-directory/Makefile

```diff
@@ -1,4 +1,4 @@
-REPO := github.com/open-olive/sidekick-controller-examplego
```

Comment:
There were a lot of references throughout the Go examples to this older example module.